templates/shared_services/firewall/terraform/locals.tf

```diff
@@ -1,30 +1,8 @@
 locals {
   core_resource_group_name = "rg-${var.tre_id}"
-  firewall_name            = "fw-${var.tre_id}"
-  firewall_diagnostic_categories_enabled = [
-    "AZFWApplicationRule",
-    "AZFWApplicationRuleAggregation",
-    "AZFWDnsQuery",
-    "AZFWFatFlow",
-    "AZFWFlowTrace",
-    "AZFWIdpsSignature",
-    "AZFWInternalFqdnResolutionFailure",
-    "AZFWNatRule",
-    "AZFWNatRuleAggregation",
-    "AZFWNetworkRule",
-    "AZFWNetworkRuleAggregation",
-    "AZFWThreatIntel"
-  ]
-  tre_shared_service_tags = {
-    tre_id                = var.tre_id
-    tre_shared_service_id = var.tre_resource_id
-  }
 
   api_driven_application_rule_collection = jsondecode(base64decode(var.api_driven_rule_collections_b64))
   api_driven_network_rule_collection     = jsondecode(base64decode(var.api_driven_network_rule_collections_b64))
 
   firewall_policy_name = "fw-policy-${var.tre_id}"
```

Comment:
I think it is now quite easy to provide this at run time as since it is set in the core it should be provided from there as a bundle variable.

Reply:
Not sure what you mean? The policy name?

Reply:
Yes, I meant the policy name. If you want it can come directly from the core deployment.

Reply:
You mean by adding `firewall_policy_id`  here: https://github.com/marrobi/AzureTRE/blob/7e532bd19b7b58d526cf84fa007060574b75ee8f/core/terraform/resource_processor/vmss_porter/locals.tf#L16 and passing through?

Can do I guess.

Reply:
Done, and tested an upgrade.